lib/obsolete.gd

```diff
@@ -531,6 +531,31 @@ DeclareOperation( "PositionFirstComponent", [IsList,IsObject] );
 
 #############################################################################
 ##
+#O  MultRowVector( <list1>, <poss1>, <list2>, <poss2>, <mul> )
+##
+##  <#GAPDoc Label="MultRowVector">
+##  <ManSection>
+##  <Oper Name="MultRowVector" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Returns>nothing</Returns>
+##
+##  <Description>
+##  The two argument version of this operation is just a synonym for
+##  <C>MultVectorLeft</C>, which calculates <A>mul</A>*<A>list1</A> in-place.
```

Comment:
This text should make it explicit that new code should not use `MultRowVector` anymore. So perhaps replace "is just a synonym" by "is an obsolete synonym", and add at the end "New code should use `MultVectorLeft` instead." or so


Moreover, this documentation section is (I think) still being included by `doc/ref/vector.xml` -- can you move it to `doc/ref/obsolete.xml` ?